nginx-ldap-auth.logrotate

```diff
@@ -1,4 +1,5 @@
 /var/log/nginx-ldap-auth/daemon.log {
+	compress
 	delaycompress
```

Comment:
`delaycompress` doesn't do anything if `compress` isn't specified.  logrotate is deletes all log content without this.